opentreemap/treemap/ecocache.py

```diff
@@ -50,12 +50,12 @@ def _get_or_compute(prefix, filter, compute_value):
 def _get_key(prefix, filter):
     if filter and (filter.filterstr or filter.displaystr):
```

Comment:
No we still need that logic you added. I'll add this comment:

```
When a filter is active, we compute ecobenefits for only a subset of map features. 
In this case we use `universal_rev` in the cache key, since any change to a map feature might cause the filter to return a different subset.
```